### PR TITLE
Use prettier

### PR DESCRIPTION
**Pull request by Emiller88**

### CODE REVIEW

File: .github/workflows/linting.yml [hunk 1]
```diff
@@ -1,18 +1,22 @@
-name: Markdown linting
-# This workflow is triggered on pushes and PRs to the repository.
-on: [push, pull_request]  
+name: Code Linting
+on:
+  push:
```
Comment (ewels) [inline .github/workflows/linting.yml:3]: Why only master?

**Review by ewels — APPROVED** [review]

File: .github/workflows/linting.yml (hunk 1 above)
Comment (Emiller88) [reply .github/workflows/linting.yml]: Probably related to https://github.com/nf-core/nf-co.re/issues/1105

Should be fixed in 42bb08dab43240e92ce5da54c23264e38eed1d1a and we can address it if that didn't work

**Comment by Emiller88** [comment]
I think someone needs to update the require statuses 
image

Let's get that addressed before merging.

**Comment by ewels** [comment]
Done ✅

**Review by ewels — APPROVED** [review]
LGTM!

**Pull request merged**